src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -73,7 +73,9 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
         description: data.description,
         start_utc_seconds:
           start.unix() || Math.floor(new Date().getTime() / 1000),
-        duration_minutes: data.duration_hours * 60 + data.duration_minutes,
+        duration_minutes: data.duration_minutes
+          ? data.duration_hours * 60 + data.duration_minutes
+          : data.duration_hours * 60,
```

Comment:
maybe this way would look better? what do you think?
```suggestion
        duration_minutes: data.duration_hours * 60 + (data.duration_minutes ?? 0)
```

Reply:
yeah i think that's better! i'll commit it